### PR TITLE
Add more animated icon assets

### PR DESCRIPTION
**Pull request by amirh**

### CODE REVIEW

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
lgtm

**Comment by yjbanov** [comment]
I'd be curious to know how much this affect analysis time.

File: packages/flutter/lib/src/material/animated_icons/animated_icons_data.dart [hunk 1]
```diff
@@ -14,11 +14,48 @@ part of material_animated_icons;
 ///
 /// Use with [AnimatedIcon] class to show specific animated icons.
 abstract class AnimatedIcons {
+
+  /// The material design add to event icon animation.
+  static const AnimatedIconData  add_event = _$add_event;
```
Comment (tvolkert) [inline packages/flutter/lib/src/material/animated_icons/animated_icons_data.dart]: Nit: s/  / / on all these

Comment (amirh) [reply packages/flutter/lib/src/material/animated_icons/animated_icons_data.dart]: done, thanks

**Comment by tvolkert** [comment]
LGTM

**Comment by cbracken** [comment]
This appears to have negatively impacted a bunch of compile_millis benchmarks.

**Comment by amirh** [comment]
I'm following up with the VM team to see if this is expected.

**Comment by Hixie** [comment]
This also had a large impact on hot reload and hot restart times, presumably because we're now pushing 1.3MB of additional code over each time.

**Pull request merged**